packages/universal-header/src/components/icons.js

```diff
@@ -30,6 +30,12 @@ const IconsContainer = styled.div`
   `}
 `
 
+const ShowOnHover = styled.div`
+`
```

Comment:
How about moving the hover selector into `ShowOnHover` and `HideOnHover` ?

such as:

```suggestion
const ShowOnHover = styled.div`
  display: none;
  &:hover {
    display: block;
  }
`
```

Is it doable ?

Reply:
The reason to put :hover in their parent is that the hover to show/hide effect should be triggered even when its parent been hoverd.
I think current method would make code a little bit dazzled.
Please let me know if you come out a better way for it.

Reply:
Our use case seems like [referring-to-other-components](https://styled-components.com/docs/advanced#referring-to-other-components).

Would it be acceptable to you ?

Reply:
Looks great.
update it in commit 7478483